src/screens/home/components/HomeView.js

```diff
@@ -185,49 +184,41 @@ const HomeView = ({ route, navigation, setLauncherContext, setBarStyle }) => {
     }
   }, [uri, client, route, mainAppFallbackURLInitialParam, navigation, session])
 
-  useEffect(
-    function handleCozyAppFallback() {
-      if (uri) {
-        const cozyAppFallbackURL = cozyAppFallbackURLInitialParam.consume()
+  useEffect(() => {
+    async function handleSecurityFlowAndCozyAppFallback() {
+      const subdomainType = client.capabilities?.flat_subdomains
+        ? 'flat'
+        : 'nested'
 
+      let navigationObject = null
+
+      if (uri) {
+        const cozyAppFallbackURL =
+          'http://drive.dev.192-168-1-65.nip.io:8080/#/folder' // cozyAppFallbackURLInitialParam.consume()
         if (cozyAppFallbackURL) {
-          setShouldWaitCozyApp(true)
```

Comment:
Are you sure you want to remove that line? this is the one that allows the HomeView to wait until CozyAppScreen is loaded